### DISCUSSION TITLE
HP is overflowing?!?!

### DISCUSSION THREAD

**Discussion by Foxy-null** [Unlucky-Life/ankimon#18] [Show and tell]
Hey Unlucky-Life, I've encountered an odd Pokémon by chance, so I thought I'd show it to you.

![image](https://github.com/Unlucky-Life/ankimon/assets/72304646/0a9e9d19-0cc0-4a4b-8ba2-4bc71d9a1373)

**Comment by Unlucky-Life** [comment]
Hey Foxy-Null thats quite weird !
After reviewing another 2 cards, did it update the hp ? => meaning correct the overflow

**Reply by Foxy-null** [reply]
Oh, I'm sorry. I closed Anki after seeing the error message; I had another thing to do at that time.

**Reply by Unlucky-Life** [reply]
Ah no worries ! (If you encounter something like this, sometimes after reviewing a round of cards it sometimes corewcts itself by reloading the variables like max hp etc)

**Comment by Foxy-null** [comment]
Hey, I've encountered the bug again. However, this bug is a little different from the previous one. The Pokémon's max HP decreased after some reviews. Moreover, that bug didn't get fixed even after several rounds elapsed.

Here are some video captures (Both videos are sped up 20 times):

https://github.com/Unlucky-Life/ankimon/assets/72304646/ec2a10c8-5111-468b-a80d-80918321a26e


https://github.com/Unlucky-Life/ankimon/assets/72304646/bc7600aa-fddc-43f4-be48-e593fcdd6346

**Reply by Unlucky-Life** [reply]
Thats weird - propably an issue with how pokemon are identified or how max hp is calculated
